### PR TITLE
Work in LFG List

### PR DESCRIPTION
**Pull request by Mesielecat**
Signed-off-by: EL_GATO <adominguezv@outlook.com>
Added clean packet of LFGLIST and missing Opcode with handle

### CODE REVIEW

**Comment by Mesielecat** [comment]
Some issue with this commits?

**Comment by funjoker** [comment]
"mdX7 requested a review from Shauren 4 days ago"

Waiting for response

**Comment by Mesielecat** [comment]
ty funjoker xD

**Comment by Mesielecat** [comment]
@Shauren status of this?

File: WowPacketParserModule.V7_0_3_22248/Parsers/LfgHandler.cs [hunk 1]
```diff
@@ -188,6 +195,14 @@ public static void HandleLfgListInviteApplicant(Packet packet)
             }
         }
 
+        [Parser(Opcode.SMSG_LFG_LIST_UPDATE_EXPIRATION)]
+        public static void HandleLfgListUpdateExpiration(Packet packet)
+        {
+            V6_0_2_19033.Parsers.LfgHandler.ReadCliRideTicket(packet, "Ticket");
+            packet.ReadTime("TimeoutTime");
+            packet.ReadByte("Type");
```
Comment (soulfrost) [inline WowPacketParserModule.V7_0_3_22248/Parsers/LfgHandler.cs]: status

Comment (Mesielecat) [reply WowPacketParserModule.V7_0_3_22248/Parsers/LfgHandler.cs]: Do you mean that type is status?

**Pull request merged**